src/frontend/utils/validators.ts

```diff
@@ -113,30 +99,27 @@ export const erPeriodeGyldig = (
             }
         }
 
-        const fomKalenderDato = kalenderDatoMedFallback(fom, TIDENES_MORGEN);
-        const tomKalenderDato = kalenderDatoMedFallback(tom, TIDENES_ENDE);
-
-        const fomDatoErFørTomDato = erFør(fomKalenderDato, tomKalenderDato);
-        const fomDatoErLikDødsfallDato = fom === person?.dødsfallDato;
+        const fomDatoErFørTomDato = !tom || isBefore(fom, tom);
```

Comment:
`!tom` betyr i praksis uendelig frem i tid

Reply:
Kunne vi ha laget en typeguard for dette?

```
function erUendelig(date: Date | undefined): date is undefined {
    return date === undefined;
}
```

Så kan den brukes som dette

```
if (!erUendelig(date)) {
    // Within this block, TypeScript knows that `date` is a `Date`
    console.log(date.getTime()); // This will not cause a TypeScript error
}
```

Reply:
Eventuelt 
```
function erEndelig(date: Date | undefined): date is Date {
    return date !== undefined;
}
```

Reply:
Usikker på hvilken verdi det gir i dette tilfellet. Her ønsker jeg bare å lage en boolsk verdi, så blir litt overkill med alt det andre kanskje?